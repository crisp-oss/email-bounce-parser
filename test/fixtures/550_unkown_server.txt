This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<paypal@bc-node.nl>: host bc-node.nl[13.168.196.14] said: 550-The mail server
    could not deliver mail to paypal@bc-node.nl.  The account 550-or domain may
    not exist, they may be blacklisted, or missing the proper dns 550 entries.
    (in reply to RCPT TO command)
