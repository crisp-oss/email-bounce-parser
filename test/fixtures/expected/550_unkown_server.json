{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<paypal@bc-node.nl>: host bc-node.nl[13.168.196.14] said: 550-The mail server could not deliver mail to paypal@bc-node.nl.  The account 550-or domain may not exist, they may be blacklisted, or missing the proper dns 550 entries. (in reply to RCPT TO command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<paypal@bc-node.nl>: host bc-node.nl[13.168.196.14] said: 550-The mail server could not deliver mail to paypal@bc-node.nl.  The account 550-or domain may not exist, they may be blacklisted, or missing the proper dns 550 entries. (in reply to RCPT TO command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": null
      },
      "label": "The mail server could not deliver mail to paypal@bc-node.nl.  The account 550-or domain may not exist, they may be blacklisted, or missing the proper dns 550 entries.",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": null,
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "paypal@bc-node.nl",
    "server": {
      "hostname": "bc-node.nl",
      "ip": "13.168.196.14",
      "port": null
    },
    "command": "RCPT TO"
  }
}